frontend/src/pages/LogsPage/LogsTable/LogDetails.tsx

```diff
@@ -215,23 +218,43 @@ export const LogDetails = ({ row, queryTerms }: Props) => {
 							Copy link
 						</Box>
 					</ButtonLink>
-				</Box>
 
-				<Box
-					display="flex"
-					alignItems="center"
-					flexDirection="row"
-					gap="16"
-				>
+					{row.original.node.secureSessionID?.length ||
+					row.original.error_object ? (
+						<Box
+							border="divider"
+							style={{ width: 1, height: 14 }}
+						/>
+					) : null}
+
+					{row.original.node.secureSessionID?.length ? (
+						<ButtonLink
+							kind="secondary"
+							onClick={(e) => {
+								e.stopPropagation()
+								navigate(
```

Comment:
Can we instead wrap `<ButtonLink />` with `<Link to />` to make it a proper link ([example](https://github.com/highlight/highlight/pull/4653/files#diff-e1afec61a24508e9b9d003b56964ed4eb6af88381cb52334fb8eed927ee2ee36R269-R277)). 

This will make it more accessible since I can hover over the element and see exactly what link I'm actually clicking on. Along with being able to right click and get this view:
<img width="320" alt="Screenshot 2023-03-22 at 9 02 15 PM" src="https://user-images.githubusercontent.com/58678/227090331-b6e4c016-f60e-448f-8dcc-a552fcf68d4a.png">

Reply:
great suggestion!